components/ProjectCard/index.js

```diff
@@ -0,0 +1,35 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function ProjectCard({ project }) {
+  return (
+    <CardContainer>
+      <Image
+        src={project.imageUrl}
+        alt={project.title}
+        width={200}
+        height={150}
+      />
+      <Wrapper>
+        <h2>{project.title}</h2>
+        <p>
+          Complexity:
+          <br />
+          {project.complexity}
```

Comment:
```suggestion
        src={imageUrl}
        alt={title}
        width={200}
        height={150}
      />
      <Wrapper>
        <h2>{title}</h2>
        <p>
          Complexity:
          <br />
          {complexity}
```
According to my comment before.